keyboards/alpha/readme.md

```diff
@@ -0,0 +1,17 @@
+# Alpha
+
+![Alpha](https://imgur.com/a/TouJ5rH)
```

Comment:
Imgur link doesn't work. 
This should be a single image, not an album, ideally.